tests/test_chart_rendering.py

```
@@ -1,483 +1,157 @@
+def test_render_chart_ascending_values():
+    """_render_chart should show ascending pattern for increasing values."""
+    values = [1, 2, 3, 4, 5, 6, 7, 8]
+    result = _render_chart(values, width=8, height=4)
+    lines = result.split("\n")
+    
+    # Check that we have the right number of lines
+    assert len(lines) == 4
+    
+    # The top line should have more empty spaces on the left (lower values)
+    # and more filled blocks on the right (higher values)
+    top_line = lines[0]
```

Ruff (0.14.14)

Local variable `top_line` is assigned to but never used

Remove assignment to unused variable `top_line`

(F841)